### PR TITLE
Fcenet readme update

### PR DESCRIPTION
**Pull request by colawyee**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/det/fcenet/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ The FCENet in MindOCR is trained on ICDAR 2015 dataset. The training results are
...
 |---------------------|----------------|---------------|------------|------------|---------------|-------------|--------------|-----------|-------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| FCENet               | D910x4-MS2.0-F | ResNet50   | ImageNet       |  81.51%     | 86.90%        | 84.12%     | 33 s/epoch   | 7 img/s      | [yaml](fce_icdar15.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/fcenet/fcenet_resnet50-43857f7f.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/fcenet/fcenet_resnet50-43857f7f-5e765611.mindir) |
+| FCENet               | D910x4-MS2.0-F | ResNet50   | ImageNet       |  81.51%     | 86.90%        | 84.12%     | 95.59 s/epoch   | 2.59  img/s      | [yaml](fce_icdar15.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/fcenet/fcenet_resnet50-43857f7f.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/fcenet/fcenet_resnet50-43857f7f-dad7dfcc.mindir) |
```
Comment (HaoyangLee) [inline configs/det/fcenet/README.md:54]:
```suggestion
| FCENet               | D910x4-MS2.0-F | ResNet50   | ImageNet       |  81.51%     | 86.90%        | 84.12%     | 95.59 s/epoch   | 10.38  img/s      | [yaml](fce_icdar15.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/fcenet/fcenet_resnet50-43857f7f.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/fcenet/fcenet_resnet50-43857f7f-dad7dfcc.mindir) |
```
日志中显示的是单卡fps，此处填写多（四）卡fps，约等于单卡fps *4

Comment (HaoyangLee) [inline configs/det/fcenet/README.md:54]: 补充一列ms/step，可从日志中的per step time获取，多个epoch取个均值

**Comment by HaoyangLee** [comment]
https://github.com/mindspore-lab/mindocr/blob/main/docs/cn/mkdocs/modelzoo_training.md

**Review by hadipash — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]
nice work.

**Comment by SamitHuang** [comment]
It would be better to combine with https://github.com/mindspore-lab/mindocr/pull/468

**Pull request merged**